src/main/java/pl/targosz/homework/InterfaceToCountParimeterAndArea/Triangle.java

```diff
@@ -0,0 +1,23 @@
+package pl.targosz.homework.InterfaceToCountParimeterAndArea;
+
+public class Triangle implements InterfaceToCountParimeterAndArea {
+    double a,b,c,h;
```

Comment:
To pole powinno być private (analogicznie w przypadku innych klas w tym commicie). Ustawić powinieneś je za pomocą konstruktora. ;)

https://1024kb.pl/nauka-java/hermetyzacja-java/